### ISSUE TITLE
🐞 - @ng-web-apis/mutation-observer is missing after update to 2.22.0

### ISSUE THREAD

**Issue by rvalitov** [TinkoffCreditSystems/taiga-ui#979]
### Which @taiga-ui/* package(s) are the source of the bug?

core

### Description

After running `npm update` from Taiga 2.21.0 to 2.22.0 and running build, I get the following error:

```
Error: The target entry-point "@taiga-ui/kit" has missing dependencies:
 - @ng-web-apis/mutation-observer
```

In `package-lock.json` I have:
```
"node_modules/@taiga-ui/cdk/node_modules/@ng-web-apis/mutation-observer": {
      "version": "1.1.0",
      "resolved": "https://registry.npmjs.org/@ng-web-apis/mutation-observer/-/mutation-observer-1.1.0.tgz",
      "integrity": "sha512-ThcsCaGCXf07xvIAFdP967MVxE74rH6dfZtoKky6RR69aMhdPz0AQuevVKIpcX4mU2P3ROpZhqP8MGhWD7DnOg==",
      "dependencies": {
        "tslib": "^1.9.0"
      },
      "peerDependencies": {
        "@angular/core": ">=6.0.0",
        "@ng-web-apis/common": ">=1.1.0"
      }
    },
```

### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by rvalitov** [comment]
Resolved by manually installing the package:
```
npm install @ng-web-apis/mutation-observer --save
```

**Comment by waterplea** [comment]
Such issues are typically resolved by removing `node_modules` and `package-lock.json` and doing clean `npm i`. Please check if that fixes it for you.

**Comment by rvalitov** [comment]
That helped, thank you!

**Comment by rvalitov** [comment]
I did it. And made a clean install. But the issue still exists. The problem is that the missing `@ng-web-apis/mutation-observer` error happens not when I run `npm install` or `npm update` - these commands are successful.

Error message appears when I run `ng serve` or `ng build`.

**Comment by rvalitov** [comment]
Full log:
```console
ng serve
- Generating browser application bundles (phase: setup)...Warning: Entry point 'angular2-text-mask' contains deep imports into 'C:/git/project-folder/node_modules/text-mask-core/dist/textMaskCore'. This is probably not a problem, but may cause the compilation of entry points to be out of order.
Compiling @angular/core : es2015 as esm2015
Compiling @angular/animations : es2015 as esm2015
Compiling @taiga-ui/cdk/utils/browser : es2015 as esm2015
Compiling @taiga-ui/cdk/enums : es2015 as esm2015
Compiling @taiga-ui/core/enums : es2015 as esm2015
Compiling @taiga-ui/i18n/enums : es2015 as esm2015
Compiling @taiga-ui/core/utils/polyfills : es2015 as esm2015
Compiling @taiga-ui/core/animations : es2015 as esm2015
Compiling @taiga-ui/kit/enums : es2015 as esm2015
Compiling @taiga-ui/i18n/interfaces : es2015 as esm2015
Compiling @taiga-ui/kit/types : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/enums : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/english : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/dutch : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/french : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/german : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/polish : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/portuguese : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/russian : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/spanish : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/turkish : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/ukrainian : es2015 as esm2015
Compiling @taiga-ui/i18n/languages/vietnamese : es2015 as esm2015
Compiling @taiga-ui/kit/utils/format : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/types : es2015 as esm2015
Compiling @angular/cdk/keycodes : es2015 as esm2015
Compiling @taiga-ui/i18n/languages : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/utils : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/types : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/interfaces : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/const : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/enums : es2015 as esm2015
Compiling @taiga-ui/addon-table/types : es2015 as esm2015
Compiling @angular/animations/browser : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/checked : es2015 as esm2015
Compiling @angular/common : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/element : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/focusable : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/for : es2015 as esm2015
Compiling @tinkoff/ng-polymorpheus : es2015 as esm2015
Compiling @angular/forms : es2015 as esm2015
Compiling @ng-web-apis/common : es2015 as esm2015
Compiling @angular/platform-browser : es2015 as esm2015
Compiling @taiga-ui/cdk/interfaces : es2015 as esm2015
Compiling @ng-web-apis/resize-observer : es2015 as esm2015
Compiling @taiga-ui/cdk/types : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/high-dpi : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/let : es2015 as esm2015
Compiling @taiga-ui/cdk/components/portal-host : es2015 as esm2015
Compiling @taiga-ui/cdk/pipes/filter : es2015 as esm2015
Compiling @taiga-ui/cdk/pipes/mapper : es2015 as esm2015
Compiling @taiga-ui/i18n/tools : es2015 as esm2015
Compiling @taiga-ui/cdk/pipes : es2015 as esm2015
Compiling @angular/router : es2015 as esm2015
Compiling @taiga-ui/cdk/constants : es2015 as esm2015
Compiling @tinkoff/ng-event-plugins : es2015 as esm2015
Compiling @taiga-ui/i18n : es2015 as esm2015
Compiling @taiga-ui/cdk/classes : es2015 as esm2015
Compiling @taiga-ui/cdk/utils/focus : es2015 as esm2015
Compiling @taiga-ui/cdk/utils/miscellaneous : es2015 as esm2015
Compiling @taiga-ui/cdk/utils/math : es2015 as esm2015
Compiling @taiga-ui/cdk/utils/format : es2015 as esm2015
Compiling @taiga-ui/cdk/decorators : es2015 as esm2015
Compiling @taiga-ui/cdk/observables : es2015 as esm2015
Compiling @taiga-ui/cdk/utils/dom : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/media : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/repeat-times : es2015 as esm2015
Compiling @taiga-ui/cdk/utils : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/drag : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/focus-trap : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/focused : es2015 as esm2015
Compiling @taiga-ui/cdk/tokens : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/validator : es2015 as esm2015
Compiling @taiga-ui/cdk/date-time : es2015 as esm2015
Compiling @ng-web-apis/intersection-observer : es2015 as esm2015
Compiling angular2-text-mask : main as commonjs
Compiling @taiga-ui/cdk/services : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/active-zone : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/auto-focus : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/input-mode : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/pressed : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/autofilled : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/droppable : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/focus-visible : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/hovered : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/obscured : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/overscroll : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/resize : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/prevent-default : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/pan : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/swipe : es2015 as esm2015
Compiling @taiga-ui/cdk/directives/zoom : es2015 as esm2015
Compiling @taiga-ui/cdk/abstract : es2015 as esm2015
Compiling @angular/cdk/platform : es2015 as esm2015
Compiling @angular/cdk/bidi : es2015 as esm2015
Compiling @taiga-ui/cdk/directives : es2015 as esm2015
Compiling @angular/cdk/collections : es2015 as esm2015
Compiling @angular/cdk/observers : es2015 as esm2015
Compiling @angular/cdk/scrolling : es2015 as esm2015
Compiling @taiga-ui/addon-table/tokens : es2015 as esm2015
Compiling @angular/cdk/a11y : es2015 as esm2015
Compiling @taiga-ui/cdk/components/dialog-host : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/tokens : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/tokens : es2015 as esm2015
Compiling @angular/platform-browser-dynamic : es2015 as esm2015
Compiling @tinkoff/ng-dompurify : es2015 as esm2015
Compiling @angular/cdk/drag-drop : es2015 as esm2015
Compiling @taiga-ui/cdk/components : es2015 as esm2015
Compiling @angular/service-worker : es2015 as esm2015
Compiling @angular/common/http : es2015 as esm2015
Compiling @taiga-ui/cdk : es2015 as esm2015
Compiling ngx-device-detector : es2015 as esm2015
Compiling @angular/platform-browser/animations : es2015 as esm2015
Compiling @fortawesome/angular-fontawesome : es2015 as esm2015
Compiling ngx-countdown : es2015 as esm2015
Compiling @taiga-ui/core/types : es2015 as esm2015
Compiling @taiga-ui/core/utils/dom : es2015 as esm2015
Compiling @taiga-ui/core/components/theme-night : es2015 as esm2015
Compiling @taiga-ui/core/observables : es2015 as esm2015
Compiling @taiga-ui/core/constants : es2015 as esm2015
Compiling @taiga-ui/core/utils/format : es2015 as esm2015
Compiling @taiga-ui/core/utils/miscellaneous : es2015 as esm2015
Compiling @taiga-ui/core/mask : es2015 as esm2015
Compiling @taiga-ui/core/interfaces : es2015 as esm2015
Compiling @taiga-ui/core/directives/mode : es2015 as esm2015
Compiling @taiga-ui/core/components/loader : es2015 as esm2015
Compiling @taiga-ui/core/utils/mask : es2015 as esm2015
Compiling @taiga-ui/core/pipes/format-phone : es2015 as esm2015
Compiling @taiga-ui/core/pipes/pluralize : es2015 as esm2015
Compiling @taiga-ui/core/utils : es2015 as esm2015
Compiling @taiga-ui/core/directives/color : es2015 as esm2015
Compiling @taiga-ui/core/directives/scroll-into-view : es2015 as esm2015
Compiling @taiga-ui/core/directives/textfield-controller : es2015 as esm2015
Compiling @taiga-ui/core/tokens : es2015 as esm2015
Compiling @taiga-ui/core/components/expand : es2015 as esm2015
Compiling @taiga-ui/core/components/primitive-year-picker : es2015 as esm2015
Compiling @taiga-ui/core/components/group : es2015 as esm2015
Compiling @taiga-ui/kit/tokens : es2015 as esm2015
Compiling @taiga-ui/kit/utils/dom : es2015 as esm2015
Compiling @taiga-ui/core/providers : es2015 as esm2015
Compiling @taiga-ui/core/services : es2015 as esm2015
Compiling @taiga-ui/core/pipes/format-number : es2015 as esm2015
Compiling @taiga-ui/core/pipes/month : es2015 as esm2015
Compiling @taiga-ui/core/components/scroll-controls : es2015 as esm2015
Compiling @taiga-ui/core/components/wrapper : es2015 as esm2015
Compiling @taiga-ui/core/pipes/calendar-sheet : es2015 as esm2015
Compiling @taiga-ui/core/components/svg : es2015 as esm2015
Compiling @taiga-ui/core/directives/described-by : es2015 as esm2015
Compiling @taiga-ui/core/pipes : es2015 as esm2015
Compiling @taiga-ui/core/components/scrollbar : es2015 as esm2015
Compiling @taiga-ui/core/components/button : es2015 as esm2015
Compiling @taiga-ui/core/components/link : es2015 as esm2015
Compiling @taiga-ui/core/directives/dropdown-controller : es2015 as esm2015
Compiling @taiga-ui/core/components/dropdown-box : es2015 as esm2015
Compiling @taiga-ui/core/components/primitive-spin-button : es2015 as esm2015
Compiling @taiga-ui/core/components/notification : es2015 as esm2015
Compiling @taiga-ui/core/directives/hint-controller : es2015 as esm2015
Compiling @taiga-ui/core/abstract : es2015 as esm2015
Compiling @taiga-ui/core/internal/primitive-year-month-pagination : es2015 as esm2015
Compiling @taiga-ui/core/directives/mask-accessor : es2015 as esm2015
Compiling @taiga-ui/core/modules/notifications : es2015 as esm2015
Compiling @taiga-ui/core/directives/hint : es2015 as esm2015
Compiling @taiga-ui/core/directives/dropdown : es2015 as esm2015
Compiling @taiga-ui/core/directives/pointer-hint : es2015 as esm2015
Compiling @taiga-ui/core/internal/svg-defs-host : es2015 as esm2015
Compiling @taiga-ui/core/components/primitive-calendar : es2015 as esm2015
Compiling @taiga-ui/core/components/tooltip : es2015 as esm2015
Compiling @taiga-ui/core/components/hints-host : es2015 as esm2015
Compiling @taiga-ui/core/components/data-list : es2015 as esm2015
Compiling @taiga-ui/core/components/calendar : es2015 as esm2015
Compiling @taiga-ui/core/components/dialog : es2015 as esm2015
Compiling @taiga-ui/core/components/error : es2015 as esm2015
Compiling @taiga-ui/core/components/hosted-dropdown : es2015 as esm2015
Compiling @taiga-ui/core/components/label : es2015 as esm2015
Compiling @taiga-ui/core/components/primitive-checkbox : es2015 as esm2015
Compiling @taiga-ui/core/components/primitive-textfield : es2015 as esm2015
Compiling @taiga-ui/core/components/root : es2015 as esm2015
Compiling @taiga-ui/core/directives/manual-hint : es2015 as esm2015
Compiling @taiga-ui/core/directives/table-mode : es2015 as esm2015
Compiling @taiga-ui/core/internal : es2015 as esm2015
Compiling @taiga-ui/core/modules : es2015 as esm2015
Compiling @taiga-ui/core/directives : es2015 as esm2015
Compiling @taiga-ui/core/components : es2015 as esm2015
Compiling @taiga-ui/kit/validators : es2015 as esm2015
Compiling @taiga-ui/kit/directives/highlight : es2015 as esm2015
Compiling @taiga-ui/kit/directives/lazy-loading : es2015 as esm2015
Compiling @taiga-ui/kit/directives/present : es2015 as esm2015
Compiling @taiga-ui/kit/directives/project-class : es2015 as esm2015
Compiling @taiga-ui/kit/directives/unfinished-validator : es2015 as esm2015
Compiling @taiga-ui/core : es2015 as esm2015
Compiling @taiga-ui/kit/classes : es2015 as esm2015
Compiling @taiga-ui/kit/utils/files : es2015 as esm2015
Compiling @taiga-ui/kit/components/radio-group : es2015 as esm2015
Compiling @taiga-ui/kit/utils/miscellaneous : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/directives/ripple : es2015 as esm2015
Compiling @taiga-ui/kit/pipes/stringify : es2015 as esm2015
Compiling @zxing/ngx-scanner : es2015 as esm2015
Compiling @taiga-ui/kit/providers : es2015 as esm2015
Compiling @taiga-ui/kit/constants : es2015 as esm2015
Compiling @taiga-ui/kit/directives/dropdown-hover : es2015 as esm2015
Compiling @taiga-ui/kit/directives/dropdown-selection : es2015 as esm2015
Compiling @taiga-ui/kit/directives/value-accessor : es2015 as esm2015
Compiling @taiga-ui/kit/directives/dropdown-context : es2015 as esm2015
Compiling @taiga-ui/kit/utils/mask : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/utils : es2015 as esm2015
Compiling @taiga-ui/kit/interfaces : es2015 as esm2015
Compiling @taiga-ui/kit/components/select-option : es2015 as esm2015
Compiling @taiga-ui/kit/abstract : es2015 as esm2015
Compiling @taiga-ui/kit/directives : es2015 as esm2015
Compiling @taiga-ui/kit/internal/primitive-calendar-range : es2015 as esm2015
Compiling @taiga-ui/kit/utils : es2015 as esm2015
Compiling @taiga-ui/kit/components/checkbox : es2015 as esm2015
Compiling @taiga-ui/kit/components/arrow : es2015 as esm2015
Compiling @taiga-ui/kit/components/radio : es2015 as esm2015
Compiling @taiga-ui/kit/components/badge : es2015 as esm2015
Compiling @taiga-ui/kit/components/calendar-month : es2015 as esm2015
Compiling @taiga-ui/kit/components/tag : es2015 as esm2015
Compiling @taiga-ui/kit/pipes/filter-by-input : es2015 as esm2015
Compiling @taiga-ui/kit/pipes/stringify-content : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/directives/elastic-sticky : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/directives/sidebar : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/directives/touchable : es2015 as esm2015
Compiling @taiga-ui/kit/components/marker-icon : es2015 as esm2015
Compiling @taiga-ui/kit/components/checkbox-block : es2015 as esm2015
Compiling @taiga-ui/kit/components/calendar-range : es2015 as esm2015
Compiling @taiga-ui/kit/components/range : es2015 as esm2015
Compiling @taiga-ui/kit/components/slider : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-phone : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-tag : es2015 as esm2015
Compiling @taiga-ui/kit/components/multi-select-option : es2015 as esm2015
Compiling @taiga-ui/kit/utils/math : es2015 as esm2015
Compiling @taiga-ui/kit/components/radio-labeled : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/constants : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/pipes : es2015 as esm2015
Compiling ngx-pipes : es2015 as esm2015
Compiling @taiga-ui/kit/internal : es2015 as esm2015
Compiling @taiga-ui/kit/pipes : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/directives : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/interfaces : es2015 as esm2015
Compiling @taiga-ui/addon-table/utils : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/validators : es2015 as esm2015
Compiling @taiga-ui/kit/components/accordion : es2015 as esm2015
Compiling @taiga-ui/kit/components/action : es2015 as esm2015
Compiling @taiga-ui/kit/components/avatar : es2015 as esm2015
Compiling @taiga-ui/kit/components/badged-content : es2015 as esm2015
Compiling @taiga-ui/kit/components/breadcrumbs : es2015 as esm2015
Compiling @taiga-ui/kit/components/checkbox-labeled : es2015 as esm2015
Compiling @taiga-ui/kit/components/combo-box : es2015 as esm2015
Compiling @taiga-ui/kit/components/data-list-wrapper : es2015 as esm2015
Compiling @taiga-ui/kit/components/field-error : es2015 as esm2015
Compiling @taiga-ui/kit/components/filter : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-inline : es2015 as esm2015
Compiling @taiga-ui/kit/components/input : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-copy : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-count : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-date : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-date-range : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-date-time : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-file : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-month : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-month-range : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-number : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-password : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-range : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-slider : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-time : es2015 as esm2015
Compiling @taiga-ui/kit/components/input-phone-international : es2015 as esm2015
Compiling @taiga-ui/kit/components/island : es2015 as esm2015
Compiling @taiga-ui/kit/components/line-clamp : es2015 as esm2015
Compiling @taiga-ui/kit/components/multi-select : es2015 as esm2015
Compiling @taiga-ui/kit/components/pagination : es2015 as esm2015
Compiling @taiga-ui/kit/components/progress : es2015 as esm2015
Compiling @taiga-ui/kit/components/radio-block : es2015 as esm2015
Compiling @taiga-ui/kit/components/radio-list : es2015 as esm2015
Compiling @taiga-ui/kit/components/select : es2015 as esm2015
Compiling @taiga-ui/kit/components/stepper : es2015 as esm2015
Compiling @taiga-ui/kit/components/tree : es2015 as esm2015
Compiling @taiga-ui/kit/components/text-area : es2015 as esm2015
Compiling @taiga-ui/kit/components/toggle : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/components/mobile-dialog : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/components/pull-to-refresh : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/components/theme-android : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/components/theme-ios : es2015 as esm2015
Compiling @taiga-ui/addon-table/components/reorder : es2015 as esm2015
Compiling @taiga-ui/addon-table/components/resizable-column : es2015 as esm2015
Compiling @taiga-ui/addon-table/components/table-pagination : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/components/card : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/components/input-card-grouped : es2015 as esm2015
Compiling @taiga-ui/addon-commerce/components/money : es2015 as esm2015
Compiling @taiga-ui/addon-mobile/internal/primitive-calendar-mobile : es2015 as esm2015
Compiling @taiga-ui/addon-table/interfaces : es2015 as esm2015
C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\entry_point_finder\targeted_entry_point_finder.js:40
                throw new Error("The target entry-point \"" + invalidTarget.entryPoint.name + "\" has missing dependencies:\n" +
                ^

Error: The target entry-point "@taiga-ui/kit" has missing dependencies:
 - @ng-web-apis/mutation-observer

    at TargetedEntryPointFinder.findEntryPoints (C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\entry_point_finder\targeted_entry_point_finder.js:40:23)
    at C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\execution\analyze_entry_points.js:29:41
    at SingleProcessExecutorSync.SingleProcessorExecutorBase.doExecute (C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\execution\single_process_executor.js:28:29)
    at C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\execution\single_process_executor.js:57:59
    at SyncLocker.lock (C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\locking\sync_locker.js:34:24)
    at SingleProcessExecutorSync.execute (C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\execution\single_process_executor.js:57:27)
    at Object.mainNgcc (C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\src\main.js:74:25)
    at Object.process (C:\git\project-folder\node_modules\@angular\compiler-cli\ngcc\index.js:29:23)
    at NgccProcessor.processModule (C:\git\project-folder\node_modules\@ngtools\webpack\src\ngcc_processor.js:175:16)
    at C:\git\project-folder\node_modules\@ngtools\webpack\src\ivy\host.js:146:18
    at C:\git\project-folder\node_modules\@ngtools\webpack\src\ivy\host.js:76:24
    at Array.map (<anonymous>)
    at Object.host.resolveModuleNames (C:\git\project-folder\node_modules\@ngtools\webpack\src\ivy\host.js:74:32)
    at actualResolveModuleNamesWorker (C:\git\project-folder\node_modules\typescript\lib\typescript.js:109497:133)
    at resolveModuleNamesWorker (C:\git\project-folder\node_modules\typescript\lib\typescript.js:109760:26)
    at resolveModuleNamesReusingOldState (C:\git\project-folder\node_modules\typescript\lib\typescript.js:109857:24)
```

**Comment by rvalitov** [comment]
My `package.json` file:
```json
{
  "name": "app",
  "version": "1.0.34",
  "scripts": {
    "ng": "ng",
    "build": "ng build",
    "test": "ng test"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^12.2.9",
    "@angular/cdk": "^12.2.9",
    "@angular/common": "^12.2.9",
    "@angular/compiler": "^12.2.9",
    "@angular/core": "^12.2.9",
    "@angular/fire": "^7.1.0",
    "@angular/forms": "^12.2.9",
    "@angular/platform-browser": "^12.2.9",
    "@angular/platform-browser-dynamic": "^12.2.9",
    "@angular/router": "^12.2.9",
    "@angular/service-worker": "^12.2.9",
    "@fortawesome/angular-fontawesome": "^0.9.0",
    "@fortawesome/fontawesome-svg-core": "^1.2.35",
    "@fortawesome/free-brands-svg-icons": "^5.15.3",
    "@fortawesome/free-regular-svg-icons": "^5.15.3",
    "@fortawesome/free-solid-svg-icons": "^5.15.3",
    "@ng-web-apis/mutation-observer": "^1.1.0",
    "@taiga-ui/addon-charts": "^2.20.0",
    "@taiga-ui/addon-commerce": "^2.20.0",
    "@taiga-ui/addon-mobile": "^2.20.0",
    "@taiga-ui/addon-table": "^2.20.0",
    "@taiga-ui/cdk": "^2.20.0",
    "@taiga-ui/core": "^2.20.0",
    "@taiga-ui/icons": "^2.20.0",
    "@taiga-ui/kit": "^2.20.0",
    "@tinkoff/ng-dompurify": "3.0.0",
    "@zxing/library": "^0.18.6",
    "@zxing/ngx-scanner": "^3.2.0",
    "compare-versions": "^4.0.0",
    "dompurify": "2.2.9",
    "firebase": "^9.1.2",
    "libphonenumber-js": "^1.9.37",
    "moment": "^2.29.1",
    "ngx-countdown": "^12.0.1",
    "ngx-device-detector": "^2.1.1",
    "ngx-pipes": "^2.7.5",
    "rxfire": "^6.0.0",
    "tslib": "^2.3.0",
    "uuid": "^8.3.2",
    "web-animations-js": "^2.3.2",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/architect": "^0.1200.0",
    "@angular-devkit/build-angular": "^12.2.9",
    "@angular/cli": "^12.2.9",
    "@angular/compiler-cli": "^12.2.9",
    "@types/dompurify": "2.2.3",
    "@types/jasmine": "~3.8.0",
    "@types/node": "^12.20.28",
    "@types/uuid": "^8.3.1",
    "firebase-tools": "^9.20.0",
    "fuzzy": "^0.1.3",
    "inquirer": "^6.2.2",
    "inquirer-autocomplete-prompt": "^1.0.1",
    "jasmine-core": "~3.8.0",
    "jsonc-parser": "^3.0.0",
    "karma": "~6.3.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.0.3",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "~1.7.0",
    "open": "^7.0.3",
    "prompt-confirm": "^2.0.4",
    "prompt-sync": "^4.2.0",
    "pwa-asset-generator": "^5.0.0",
    "semver-compare": "^1.0.0",
    "typescript": "~4.3.5"
  }
}
```

**Comment by waterplea** [comment]
That is strange, because MutationObserver has been there for a very long time and we didn't touch it for a long time as well.

**Comment by waterplea** [comment]
Managed to reproduce it with your `package.json` and a clean project. Will try to look into it.

**Comment by devindi** [comment]
Hello. I have pretty same error:
"An unhandled exception occurred: The target entry-point "@taiga-ui/addon-table" has missing dependencies:
 - @angular/cdk/drag-drop
"

**Comment by waterplea** [comment]
> Hello. I have pretty same error:
> "An unhandled exception occurred: The target entry-point "@taiga-ui/addon-table" has missing dependencies:
>  - @angular/cdk/drag-drop
> "

That's a different one. It's a peer dependency which you should install yourself. @vladimirpotekhin do we add it in `ng add taiga` if a person chose to add table addon? It's also needed for mobile addon

**Comment by vladimirpotekhin** [comment]
> > Hello. I have pretty same error:
> > "An unhandled exception occurred: The target entry-point "@taiga-ui/addon-table" has missing dependencies:
> > 
> > * @angular/cdk/drag-drop
> >   "
> 
> That's a different one. It's a peer dependency which you should install yourself. @vladimirpotekhin do we add it in `ng add taiga` if a person chose to add table addon? It's also needed for mobile addon

No, this dependency are not being added with the ng add. I'll fix it

**Comment by waterplea** [comment]
Hey @rvalitov could you give it a shot in the new version (2.26.0)? I believe it should be fixed now, packages were missing from peer deps this probably caused them to stay in `node_modules` of `cdk` and not bubble in the top level `node_modules`.

**Comment by rvalitov** [comment]
@waterplea yes, it works and fixed with 2.26.0. Thank you!
